bin/next-build

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env node
```

Comment:
Do you really **need** to change this? While I have never seen anyone running `next-build` or `next-dev` commands (instead of `next build` / `next dev`, where `next` command gets executed), this seems to be a breaking change to me, and given the files locate in the `bin` folder, it makes sense to be able to execute them (so shebang is necessary).

Reply:
I don't like this change their, but I haven't found a way to pass the `--inspect` flag to `#!/usr/bin/env node` otherwise.

Reply:
@auchenberg having this shebang doesn't require you use them. Just leave shebang even if you are going to run them from `next` using explicit `node --inspect next-build`.